spec/spec_helper_acceptance.rb

```diff
@@ -12,13 +12,6 @@
     # refresh check if cache needs refresh on next yum command
     on host, 'yum clean expire-cache'
   end
-
-  # In Puppet 7 the locale ends up being C.UTF-8 if it isn't passed.
```

Comment:
Please also drop this bit:
https://github.com/theforeman/puppet-foreman/blob/a71cb9e4b5b26549efe0f78f941ceaca00e99b20/.sync.yml#L10-L11=